config/routes.rb

```diff
@@ -2,6 +2,7 @@
   namespace :api do
     namespace :v1 do
       resources :articles, only: [:index, :create, :update], constraints: { format: 'json' }
+      resources :subscriptions, only: [:new, :create, :index]
```

Comment:
same here
why do you need the new action?

Reply:
deleted